app/controllers/messages_controller.rb

```diff
@@ -71,7 +74,7 @@ def set_message
     def message_params
       p = params.require(:message).permit(:receiver_id, :body)
       if current_user
-        p[:sender_id] = current_user.id
+        p[:sender_id] = session[:user_id]
```

Comment:
Why did you change this?

Reply:
I was getting strange behavior and it fixed it. Are there any side effects with this? If so I can change back

Reply:
I don't know, but if it works, it works.